core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1802,19 +1802,34 @@ class SparkContext(config: SparkConf) extends Logging {
    * Adds a JAR dependency for all tasks to be executed on this `SparkContext` in the future.
    * @param path can be either a local file, a file in HDFS (or other Hadoop-supported filesystems),
    * an HTTP, HTTPS or FTP URI, or local:/path for a file on every worker node.
+   * If addToCurrentClassLoader is true, attempt to add the new class to the current threads' class
```

Comment:
Add to doc that already loaded urls will have no effect if a url is already present.